src/openrct2-ui/interface/Theme.cpp

```diff
@@ -355,41 +355,40 @@ void UITheme::RemoveEntry(rct_windowclass windowClass)
     }
 }
 
-json_t* UITheme::ToJson() const
+json_t UITheme::ToJson() const
 {
     // Create entries
-    json_t* jsonEntries = json_object();
+    json_t jsonEntries;
     for (const UIThemeWindowEntry& entry : Entries)
     {
         const WindowThemeDesc* wtDesc = GetWindowThemeDescriptor(entry.WindowClass);
         if (wtDesc == nullptr)
         {
             return nullptr;
         }
-        json_object_set_new(jsonEntries, wtDesc->WindowClassSZ, entry.ToJson());
+        jsonEntries[wtDesc->WindowClassSZ] = entry.ToJson();
     }
 
     // Create theme object
-    json_t* jsonTheme = json_object();
-    json_object_set_new(jsonTheme, "name", json_string(Name.c_str()));
-    json_object_set_new(jsonTheme, "entries", jsonEntries);
-
-    json_object_set_new(jsonTheme, "useLightsRide", json_boolean(Flags & UITHEME_FLAG_USE_LIGHTS_RIDE));
-    json_object_set_new(jsonTheme, "useLightsPark", json_boolean(Flags & UITHEME_FLAG_USE_LIGHTS_PARK));
-    json_object_set_new(
-        jsonTheme, "useAltScenarioSelectFont", json_boolean(Flags & UITHEME_FLAG_USE_ALTERNATIVE_SCENARIO_SELECT_FONT));
-    json_object_set_new(jsonTheme, "useFullBottomToolbar", json_boolean(Flags & UITHEME_FLAG_USE_FULL_BOTTOM_TOOLBAR));
+    json_t jsonTheme = {
+        {"name", Name},
+        {"entries", jsonEntries},
+        {"useLightsRide", 0 != (Flags & UITHEME_FLAG_USE_LIGHTS_RIDE)},
```

Comment:
I would prefer not to use 'yoda coding', i.e.:
```suggestion
        {"useLightsRide", (Flags & UITHEME_FLAG_USE_LIGHTS_RIDE) != 0},
```